### ISSUE TITLE
Put the date info e.g. "April 5, 9:00" as a subline in the checkbox

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/mail#6249]

**Comment by jancborchardt** [comment]
Yup, as @nimishavijay mocked up, it would look much nicer like this, with a subline in `color-text-maxcontrast` (additionally the radio button could be vertically aligned with both text lines):
![image](https://user-images.githubusercontent.com/925062/163428367-d677eccd-1e7d-4bf4-8564-f0e6a3dc5676.png)

This might be rather an issue for the Vue components though?
